### ISSUE TITLE
Localserver up error

### ISSUE THREAD

**Issue by jzdayz** [Netflix/conductor#1685]
# Setup
1. git clone https://github.com/Netflix/conductor.git
2. git checkout tags/v2.27.0 -b v2.27.0.RELEASE
3. cd server
4. ../gradlew server

i got this error 
```log
▶ ../gradlew server
Parallel execution is an incubating feature.

> Configure project :
Inferred project: conductor, version: 2.28.0-SNAPSHOT
Publication nebula not found in project :.

> Task :conductor-server:server
Using config file: src/main/resources/server.properties
Using log4j config src/main/resources/log4j.properties
1    [main] INFO  com.netflix.conductor.bootstrap.ModulesProvider  - Starting conductor server using in memory data store.
13   [main] WARN  com.netflix.conductor.bootstrap.ModulesProvider  - Starting locking module using Noop Lock.
658  [main] INFO  com.netflix.conductor.core.execution.tasks.SystemTaskWorkerCoordinator  - Adding the queue for system task: HTTP
659  [main] INFO  com.netflix.conductor.contribs.http.HttpTask  - HttpTask initialized...
663  [main] INFO  com.netflix.conductor.core.execution.tasks.SystemTaskWorkerCoordinator  - Adding the queue for system task: KAFKA_PUBLIS
663  [main] INFO  com.netflix.conductor.contribs.kafka.KafkaPublishTask  - KafkaTask initialized...
664  [main] INFO  com.netflix.conductor.core.execution.tasks.SystemTaskWorkerCoordinator  - Adding the queue for system task: JSON_JQ_TRANSFORM
Exception in thread "main" java.lang.NoClassDefFoundError: org/elasticsearch/client/Client
        at com.netflix.conductor.elasticsearch.ElasticSearchModule.configure(ElasticSearchModule.java:28)
        at com.google.inject.AbstractModule.configure(AbstractModule.java:62)
        at com.google.inject.spi.Elements$RecordingBinder.install(Elements.java:340)
        at com.google.inject.spi.Elements.getElements(Elements.java:110)
        at com.google.inject.util.Modules$OverrideModule.configure(Modules.java:177)
        at com.google.inject.AbstractModule.configure(AbstractModule.java:62)
        at com.google.inject.spi.Elements$RecordingBinder.install(Elements.java:340)
        at com.google.inject.spi.Elements.getElements(Elements.java:110)
        at com.google.inject.internal.InjectorShell$Builder.build(InjectorShell.java:138)
        at com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:104)
        at com.google.inject.Guice.createInjector(Guice.java:99)
        at com.google.inject.Guice.createInjector(Guice.java:73)
        at com.netflix.conductor.bootstrap.Main.main(Main.java:45)
Caused by: java.lang.ClassNotFoundException: org.elasticsearch.client.Client
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
        ... 13 more

```

**Comment by accfcx** [comment]
I'm sorry, how do you solve this problem? I have no idea.

**Comment by vargaendre** [comment]
I had the same issue, but with docker run. How could it be solved?

**Comment by elisherer** [comment]
Same here (on Windows 10.0.17763.805 Enterprise)

**Comment by jzdayz** [comment]
@vargaendre  @elisherer  @accfcx  

> Obviously , it's a bug ,so i go back to 2.26.1 ,and it's working.

**Comment by jzdayz** [comment]
I think must fix it,so l reopened.

**Comment by Baural** [comment]
Hi I have same issue.
I downgraded jvm from java 11 to java 8
I downgraded gradle from 6.0.1 to 4.8.1
And switched tags to 2.27

However it is not helped to fix and run server in my Mac

> Task :conductor-redis-persistence:compileJava
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :conductor-server:server FAILED
Using config file: src/main/resources/server.properties
Using log4j config src/main/resources/log4j.properties
0    [main] INFO  com.netflix.conductor.bootstrap.ModulesProvider  - Starting conductor server using in memory data store.
9    [main] WARN  com.netflix.conductor.bootstrap.ModulesProvider  - Starting locking module using Noop Lock.
218  [main] INFO  com.netflix.conductor.core.execution.tasks.SystemTaskWorkerCoordinator  - Adding the queue for system task: HTTP
218  [main] INFO  com.netflix.conductor.contribs.http.HttpTask  - HttpTask initialized...
221  [main] INFO  com.netflix.conductor.core.execution.tasks.SystemTaskWorkerCoordinator  - Adding the queue for system task: KAFKA_PUBLISH
221  [main] INFO  com.netflix.conductor.contribs.kafka.KafkaPublishTask  - KafkaTask initialized...
222  [main] INFO  com.netflix.conductor.core.execution.tasks.SystemTaskWorkerCoordinator  - Adding the queue for system task: JSON_JQ_TRANSFORM
Exception in thread "main" java.lang.NoClassDefFoundError: org/elasticsearch/client/Client
        at com.netflix.conductor.elasticsearch.ElasticSearchModule.configure(ElasticSearchModule.java:28)
        at com.google.inject.AbstractModule.configure(AbstractModule.java:62)
        at com.google.inject.spi.Elements$RecordingBinder.install(Elements.java:340)
        at com.google.inject.spi.Elements.getElements(Elements.java:110)
        at com.google.inject.util.Modules$OverrideModule.configure(Modules.java:177)
        at com.google.inject.AbstractModule.configure(AbstractModule.java:62)
        at com.google.inject.spi.Elements$RecordingBinder.install(Elements.java:340)
        at com.google.inject.spi.Elements.getElements(Elements.java:110)
        at com.google.inject.internal.InjectorShell$Builder.build(InjectorShell.java:138)
        at com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:104)
        at com.google.inject.Guice.createInjector(Guice.java:99)
        at com.google.inject.Guice.createInjector(Guice.java:73)
        at com.netflix.conductor.bootstrap.Main.main(Main.java:45)
Caused by: java.lang.ClassNotFoundException: org.elasticsearch.client.Client
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 13 more

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':conductor-server:server'.
> Process 'command '/Library/Java/JavaVirtualMachines/jdk1.8.0_211.jdk/Contents/Home/bin/java'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
See https://docs.gradle.org/4.8.1/userguide/command_line_interface.html#sec:command_line_warnings

**Comment by manikantanallagatla** [comment]
I am facing the issue on Mac as well.

**Comment by chinazhaoke97zk** [comment]
I am facing the issue on Mac as well.How could it be solved?

**Comment by manikantanallagatla** [comment]
I solved it by going to  v2.25.8

**Comment by apanicker-nflx** [comment]
This might be related to changes here - https://github.com/Netflix/conductor/commit/167180ad517ef8d65679e2094ad3a63f4920cd5b#diff-90410c75a439e8a737f3bdc2c3794922R6-R9. 
We will address this and release a fix soon.

**Comment by slipmp** [comment]
@manikantanallagatla your suggestion worked perfectly:
After checking out tag/v2.25.8 then executing docker-compose up for serverAndUI it worked! I just want to study the platform, latest version was desired but not required for that purpose.

**Comment by slipmp** [comment]
Actually server docker is exiting with code 1 - error. 
I've tried tag/2.24.2 and all dockers work.

**Comment by apanicker-nflx** [comment]
This has been fixed in [v2.27.2](https://github.com/Netflix/conductor/releases/tag/v2.27.2). Apologies for the inconvenience.
